spec/system/admin_customize_themes_spec.rb

```diff
@@ -35,15 +35,13 @@
 
       visit("/admin/customize/themes/#{theme.id}/common/head_tag/edit")
 
-      ace_content = find(".ace_content")
-      expect(ace_content.text).to eq("console.log('test')")
+      expect(find(".ace_content").text).to eq("console.log('test')")
```

Comment:
I think we need to use Capybara matches here so that we can rely on the auto wait behaviour.

Reply:
This was not the error I had though, I had a node stale error I think, I don't think it has ever been flakey since.